app/src/questions/models.py

```diff
@@ -5,3 +5,5 @@
 class QA(models.Model):
     question = models.CharField(_('Question'), max_length=150)
     answer = models.CharField(_('Answer'), max_length=500)
+    # will be sorted in descending order
```

Comment:
You should put this comment in `help_text` for `priority` field, therefore it'll appear in form. Maybe it should be slightly changed to e.g. `Questions are sorted in descending order of priorities`. Remember to re-create migration.